### PR TITLE
fix(DB/Creature): Pacifies Theramore Combat Dummies

### PR DESCRIPTION
**Pull request by Azcobu**
## Changes Proposed:
-  Makes the Theramore Combat Dummies into inert targets again, and stops them chasing people.

## Issues Addressed:

- Closes https://github.com/azerothcore/azerothcore-wotlk/issues/6321

## SOURCE:

This is not a controversial change.

## Tests Performed:

- Keira3 generated the SQL. Executed with no problems.
- Then checked in-game, dummies behaved properly.

## How to Test the Changes:

1. Attack a Theramore dummy. If it chases you or moves at all, this change did not work.

## Known Issues and TODO List:

N/A.

## How to Test AzerothCore PRs
 
When a PR is ready to be tested, it will be marked as **[WAITING TO BE TESTED]**.

You can help by testing PRs and writing your feedback here on the PR's page on GitHub. Follow the instructions here:

http://www.azerothcore.org/wiki/How-to-test-a-PR

**REMEMBER**: when testing a PR that changes something **generic** (i.e. a part of code that handles more than one specific thing), the tester should not only check that the PR does its job (e.g. fixing spell XXX) but **especially** check that the PR does not cause any regression (i.e. introducing new bugs).

**For example**: if a PR fixes spell X by changing a part of code that handles spells X, Y, and Z, we should not only test X, but **we should test Y and Z as well**.

### CODE REVIEW

File: data/sql/updates/pending_db_world/rev_1623675094427536149.sql [hunk 1]
```diff
@@ -0,0 +1,4 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1623675094427536149');
+
+-- Makes Theramore Combat Dummies boring again.
+UPDATE `creature_template` SET `mechanic_immune_mask` = 32, `flags_extra` = 262144, `ScriptName` = 'npc_training_dummy' WHERE (`entry` = 4952);
```
Comment (Kitzunu) [inline data/sql/updates/pending_db_world/rev_1623675094427536149.sql:4]: https://www.azerothcore.org/wiki/sql-standards#flags-amp-bits

Comment (Azcobu) [reply data/sql/updates/pending_db_world/rev_1623675094427536149.sql]: Cheers, rewritten to meet standard.

Comment (Azcobu) [reply data/sql/updates/pending_db_world/rev_1623675094427536149.sql]: Actually, just tested this and the combat dummies are dying if you hit them hard enough, so needs more work still.

Comment (Azcobu) [reply data/sql/updates/pending_db_world/rev_1623675094427536149.sql]: Was working with refreshed DB for other testing, so the above changes weren't included. Dummies running the new changes don't die, but they do take half-health damage, which the dummies I copied their settings from (Expert's Training Dummy in Org) do not, so still missing a difference somewhere.

**Review by FrancescoBorzi — APPROVED** [review]

**Pull request merged**